src/Drupal/Translatable/Content/FieldCollectionTranslatable.php

```diff
@@ -51,8 +51,33 @@ public function getSourceLanguage() {
    */
   public function getTargetEntity($targetLanguage) {
     if (!isset($this->targetEntities[$targetLanguage]) || empty($this->targetEntities[$targetLanguage])) {
+      $target = $this->getRawEntity($this->entity);
```

Comment:
These changes are a reversion to a previous state. 
I got confused by the "field is translatable" setting causing all kinds of havoc.
If I don't revert these then I need to comment out a whole lot of other tests...